embrace-android-sdk/src/main/java/io/embrace/android/embracesdk/internal/injection/ModuleInitBootstrapper.kt

```diff
@@ -350,10 +352,17 @@ internal class ModuleInitBootstrapper(
                     }
 
                     postInit(NativeFeatureModule::class) {
+                        val configService = configModule.configService
                         serviceRegistry.registerServices(
                             lazy { nativeFeatureModule.nativeThreadSamplerService }
                         )
-                        nativeFeatureModule.nativeCrashHandlerInstaller?.install()
+
+                        if (configService.autoDataCaptureBehavior.isNativeCrashCaptureEnabled()) {
+                            val worker = workerThreadModule.backgroundWorker(Worker.Background.IoRegWorker)
+                            worker.submit {
```

Comment:
We are passing a worker to the nativeCrashHandlerInstaller, and `install()` happens within that worker already.